crates/sui-storage/src/event_store/sql.rs

```diff
@@ -142,7 +154,10 @@ impl SqlEventStore {
             let mut interval = tokio::time::interval(cleanup_interval);
             loop {
                 interval.tick().await;
+                let start = Instant::now();
                 info!("Running SQLite WAL truncation...");
+                let _guard = self.query_lock.write().await;
+                info!("Acquired query_lock for write after {:?}", start.elapsed());
```

Comment:
Can we log the results from `.execute("PRAGMA wal_checkpoint(TRUNCATE)")` ? (I can't comment there) 
The result from `PRAGMA wal_checkpoint(TRUNCATE)` would tell us whether the checkpointing succeeds

Reply:
I tried - not sure if it will be useful